hack/ci/e2e-k8s.sh

```diff
@@ -129,6 +129,9 @@ create_cluster() {
     ;;
   esac
 
+# Default Log level for all components in test clusters
+KIND_CLUSTER_LOG_LEVEL=${KIND_CLUSTER_LOG_LEVEL:-4}
```

Comment:
TODO: consider switching this to something that matches the rest of CI, if we have it. (kube-up?)

Reply:
```
hack/local-up-cluster.sh:LOG_LEVEL=${LOG_LEVEL:-3}
```
 or
```
cluster/gce/config-test.sh:TEST_CLUSTER_LOG_LEVEL=${TEST_CLUSTER_LOG_LEVEL:---v=4}
```